docs/reference/aggregations/metrics/geobounds-aggregation.asciidoc

```diff
@@ -84,6 +85,18 @@ The response for the above aggregation:
 
 The Geo Bounds Aggregation is also supported on `geo_shape` fields.
 
+If <<geo-bounds-wrap-longitude,`wrap_longitude`>> is set to `true`
+(the default), the bounding box can overlap the international date line. If
+these shapes also include `geo_shape` fields, the `geo_bounds` aggregation
```

Comment:
I think this behavior applies equally to just points, so I think we can remove this `if`.

e.g.:

```
If <<geo-bounds-wrap-longitude,`wrap_longitude`>> is set to `true`
(the default), the bounding box can overlap the international date line and returns a bounds where the `top_left` longitude is larger than the `top_right` longitude.
```